purebred.spec

```diff
@@ -80,29 +99,41 @@ This package provides the Haskell %{pkg_name} library development files.
 %install
 %ghc_lib_install
 %ghc_fix_rpath %{pkgver}
+mv %{buildroot}%{_ghclicensedir}/{,ghc-}%{name}
+
+install -p -m 0644 -D configs/config.hs %{buildroot}%{_datadir}/purebred/config.hs
 
 
 %check
 %cabal_test
 
 
-%post devel
+%post -n ghc-%{name}-devel
 %ghc_pkg_recache
 
 
-%postun devel
+%postun -n ghc-%{name}-devel
 %ghc_pkg_recache
 
 
-%files -f %{name}.files
+%files
+
+%files basic
+%license LICENSE
+%doc README.md
+%{_bindir}/%{name}
+
+
+%files -n ghc-%{name} -f ghc-%{name}.files
 %license LICENSE
 
 
-%files devel -f %{name}-devel.files
-%doc README.html README.md
-%{_bindir}/%{pkg_name}
+%files -n ghc-%{name}-devel -f ghc-%{name}-devel.files
+%doc README.md
 
+%files config
+%{_datadir}/purebred/config.hs
```

Comment:
Is this the default config?  If so wouldn't it be better to ship as part of base package?

Reply:
Again... peeked at the xmonad package as an example. I suppose you can still install `purebred-base` without the config and operate the client. Especially if the default key bindings are all you need. No hard feelings here either, tho arguably we could ship different config flavours for key bindings (e.g. vim-like, mutt-like, etc).

Reply:
overkill IMO.  but i guess the option is there.